backends/postgres/postgres_backend.go

```diff
@@ -271,15 +271,6 @@ func (p *PgBackend) Enqueue(ctx context.Context, job *jobs.Job) (jobID string, e
 	// Rollback is safe to call even if the tx is already closed, so if
 	// the tx commits successfully, this is a no-op
 	defer func(ctx context.Context) { _ = tx.Rollback(ctx) }(ctx) // rollback has no effect if the transaction has been committed
-
-	// Make sure RunAfter is set to a non-zero value if not provided by the caller
-	// if already set, schedule the future job
-	now := time.Now().UTC()
-	if job.RunAfter.IsZero() {
-		p.logger.Debug("RunAfter not set, job will run immediately after being enqueued")
-		job.RunAfter = now
-	}
-
```

Comment:
The schema for `neoq_jobs` already defaults to `now()`; this Go code was never necessary.